components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.html

```diff
@@ -11,7 +11,7 @@
     [ngClass]="{'active': selectionCountActive }">{{ selectionCount }}</span>
   <chef-icon *ngIf="dropdownCaretVisible" aria-hidden>keyboard_arrow_down</chef-icon>
 </button>
-<chef-click-outside omit="dropdown-label" (clickOutside)="handleEscape()">
+<chef-click-outside omit="dropdown-label" (clickOutside)="handleEscape()" *ngIf="dropdownActive">
```

Comment:
Added this in https://github.com/chef/automate/pull/2686/commits/eb514009556f8c76016c311a5560fed841abb586 to prevent `handleEscape()` from being called whenever you click somewhere in the app:

<details>
  <summary>Screenshot:</summary>
  <img src="https://user-images.githubusercontent.com/479121/73023050-f8417300-3de7-11ea-9ce1-ae75c6d1e797.gif" />
</details>


Which fixed issue preventing projects from being added to teams:

<details>
  <summary>Screenshot:</summary>
  <img src="https://user-images.githubusercontent.com/479121/73023132-1c9d4f80-3de8-11ea-88b8-c246f4d527f5.gif" />
</details>

Reply:
cool, thank you!!